tests/js/client/load-balancing/load-balancing-transactions-noauth-cluster.js

```diff
@@ -250,6 +254,69 @@ function TransactionsSuite () {
       }
     },
 
+    testUseTransaction: function () {
+      let trx;
+      try {
+        const obj = { collections: { read: cn } };
+        let result = sendRequest('POST', "/_api/transaction/begin", obj, true);
+        assertEqual(result.status, 201);
+        assertFalse(result.body.result.id === undefined);
+        trx = result.body.result.id;
+        
+        // use trx on different coord to run a query
+        const query = `FOR doc IN ${cn} RETURN doc`;
+        result = sendRequest('POST', "/_api/cursor", { query }, false);
+        assertFalse(result.body.error);
+        assertEqual(10, result.body.result.length);
```

Comment:
This tests requesting a cursor from a different coordinator, so it invokes the redirect. I think in addition we can also verify in the response that something was redirected. The HTTP response should send a special header to indicate this, but I don't know its name from the top of my head.
We should also add a test that tries to request the cursor from the correct coordinator (i.e. no redirect), and then verify that the header wasn't set.